src/domains/idpay-app/api/idpay_timeline/openapi.timeline.yml.tpl

```diff
@@ -0,0 +1,301 @@
+                iban: string
+                idTrxIssuer: string
+                idTrxAcquirer: string
+        '401':
+          description: Authentication failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '404':
+          description: The requested ID was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '429':
+          description: Too many Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+components:
+  schemas:
+    TimelineDTO:
+      type: object
+      required:
+        - lastUpdate
+        - operationList
+      properties:
+        lastUpdate:
+          type: string
+          description: date of the last update
+          format: date-time
+        operationList:
+          type: array
+          items:
+            $ref: '#/components/schemas/OperationDTO'
+          description: the list of transactions and operations of an initiative of a citizen
+    DetailOperationDTO:
+      type: object
+      required:
+        - operationId
+        - operationType
+      properties:
+        operationId:
+          type: string
+        operationType:
+          enum:
+            - PAID_REFUND
+            - TRANSACTION
+            - REVERSAL
+            - ADD_IBAN
+            - ADD_INSTRUMENT
+            - DELETE_INSTRUMENT
+            - ONBOARDING
+          type: string
+        hpan:
+          type: string
+        amount:
+          type: string
+        accrued:
+          type: string
+        operationDate:
+          type: string
+          format: date-time
+        circuitType:
+          enum:
+            - '00'
+            - '01'
+            - '02'
+            - '03'
+            - '04'
+            - '05'
+            - '06'
+            - '07'
+            - '08'
+            - '09'
+            - '10'
+          type: string
+          description: '00-> Bancomat, 01->Visa, 02->Mastercard, 03->Amex, 04->JCB, 05->UnionPay, 06->Diners, 07->PostePay, 08->BancomatPay, 09->Satispay, 10->PrivateCircuit'
+        channel:
+          type: string
+        iban:
+          type: string
+        idTrxIssuer:
+          type: string
+        idTrxAcquirer:
+          type: string
+    OperationDTO:
+      type: object
+      required:
```

Comment:
Please use https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/#oneof and create different types with all the required properties.

Reply:
OpenAPI updated with commit -> https://github.com/pagopa/cstar-infrastructure/pull/452/commits/ebd6b43f135620844e30ffb7114b3fd1f63a3a62